.github/workflows/actions.yml

```diff
@@ -295,6 +304,19 @@ jobs:
           cmake -G"Ninja" -DRULE_LAUNCH_COMPILE=ccache -DCMAKE_OSX_DEPLOYMENT_TARGET=10.10 -DSUPERNOVA=ON -DCMAKE_BUILD_TYPE=Release $EXTRA_CMAKE_FLAGS .. --debug-output
       - name: build
         run: cmake --build $BUILD_PATH --config Release --target install
+      - name: set artifact filename
+        if: matrix.artifact-suffix
+        run: |
+          # set SC_VERSION to tag (if present) or to commit SHA
+          SC_VERSION=
+          if [[ "$GITHUB_REF" == refs/tags/* ]]; then 
+            FULL_TAG=${GITHUB_REF#refs/tags/}
+            TAG=${FULL_TAG##Version-}
+            SC_VERSION=$TAG
+          else
+            SC_VERSION=$GITHUB_SHA
+          fi
+          echo "ARTIFACT_FILE=SuperCollider-$SC_VERSION-${{ matrix.artifact-suffix }}.zip" >> $GITHUB_ENV
```

Comment:
same comments above re: variables. is it not possible to define a step once and use it in multiple places? if not, we should try to move some of these things into encapsulated scripts to avoid code duplication

Reply:
I'm not aware of a way to do this, but I'm still new to GHA :)
That's pretty much the only step that's copied verbatim. I know code duplication is bad, but working on travis I really disliked wondering in which file certain variable is defined. I'd vote to keep this as is, if this is the only place with exact code duplication.

Reply:
Clarification: we could of course have a bash script somewhere that we load here, but I'm not aware of a way to define a step here in the yaml file and use it in multiple places.

Reply:
> That's pretty much the only step that's copied verbatim.

oh okay, not as big a deal then

> I know code duplication is bad, but working on travis I really disliked wondering in which file certain variable is defined.

yeah, i agree, i never liked it either

Reply:
I found a way to factor this out! Update to this PR coming soon